pkg/build/strategies/sti/sti.go

```diff
@@ -323,15 +323,18 @@ func (b *STI) Exists(config *api.Config) bool {
 		return false
 	}
 
-	// can only do incremental build if runtime image exists, so always pull image
-	previousImageExists, _ := b.docker.IsImageInLocalRegistry(config.Tag)
-	if !previousImageExists || config.ForcePull {
-		if image, _ := b.incrementalDocker.PullImage(config.Tag); image != nil {
-			previousImageExists = true
-		}
+	policy := config.PreviousImagePullPolicy
+	if len(policy) == 0 {
+		policy = api.DefaultPreviousImagePullPolicy
+	}
```

Comment:
@soltysh I think defaulter will be also on my list for the future refactoring :-) but currently we have just PullPolicy we have to default and don't want to spend a lot of time refactoring code just to default one option ;-)